pkg/vectorstore/chromem/chromem.go

```diff
@@ -4,14 +4,17 @@ import (
 	"context"
 	"log/slog"
 	"maps"
-	"runtime"
 	"strconv"
 
 	"github.com/google/uuid"
+	"github.com/gptscript-ai/knowledge/pkg/env"
 	vs "github.com/gptscript-ai/knowledge/pkg/vectorstore"
 	"github.com/philippgille/chromem-go"
 )
 
+// EmbeddingParallelThread can be set as an environment variable to control the number of parallel API calls to create embedding for documents. Default is 100
+const EmbeddingParallelThread = "EMBEDDING_PARALLEL_THREAD"
```

Comment:
Last one, I promise :)

```suggestion
const VsChromemEmbeddingParallelThread = "VS_CHROMEM_EMBEDDING_PARALLEL_THREAD"
```

Reply:
Addressed!